DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/credit-and-debit-cards/visa/visa-status-what-does-this-mean-.md

## Declined
If a Visa payment of a customer receives the status _declined_, it may be for several reasons. Only the customer can contact their credit card issuer to inquire as to why the credit card payment has been _declined_.

> For questions about credit card payments and/or a payment status (of a credit card transaction), email our Support Team at <support@multisafepay.com>

Reviewer: ```suggestion
For questions about credit card payments and transaction statuses, email the Support Team at <support@multisafepay.com>
```

In most cases the declined reason is shown behind the declined status within the transaction in your MultiSafepay Control. 

| Transaction status | Reason | Description |
------- |----------|---------|

> Declined | transaction not allowed | The fraud filter of MultiSafepay is declining the transaction. Email our Support Team at <support@multisafepay.com> for further information.

Reviewer: ```suggestion
Declined | Transaction not allowed | The MultiSafepay fraud filter declined the transaction. For more information, email the Support Team at <support@multisafepay.com>
```
